eng/jacoco-test-coverage/pom.xml

```diff
@@ -80,7 +80,7 @@
     <dependency>
```

Comment:
Did you run the update script? I thought this was also referenced in our spotbugs aggregate report pom.

Reply:
Yes. I ran the update script

Reply:
python eng\versioning\update_versions.py --update-type all --build-type client

Reply:
Now I'm confused because I can see it here:
https://github.com/Azure/azure-sdk-for-java/blob/master/eng/spotbugs-aggregate-report/pom.xml#L158

Reply:
I asking Jim for the solution. Maybe the script doesn't work properly?